Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = ["crates/eth-rpc", "crates/core"]
 resolver = "2"
 
 [workspace.package]
+version = "0.1.0"
```

Comment:
I think we should manage our version from workspace, and all other packages can share the same version.

Lemme know if you think otherwise.